app/utils/hentPath.ts

```diff
@@ -1,26 +1,31 @@
 import { ESteg } from '~/typer/felles';
 import { EYtelse } from '~/typer/ytelse';
 
+const URL_BARNETRYGD = '/ba';
+const URL_KONTANTSTØTTE = '/ks';
+
 export const hentPathForYtelse = (ytelse: EYtelse) => {
   switch (ytelse) {
     case EYtelse.BARNETRYGD:
-      return '/ba';
+      return URL_BARNETRYGD;
     case EYtelse.KONTANTSTØTTE:
-      return '/ks';
+      return URL_KONTANTSTØTTE;
     default:
       return '/';
```

Comment:
Gjerne kom med innspill på hva som bør være default på begge disse switch casene

Reply:
Hvis "/" er siden for å velge stønad så tenker jeg det funker bra inntil videre, iallefall.

Reply:
Kanskje sende til feilsiden? 🤔

Reply:
Hmm litt usikker på hva som er best her

Trenger man egentlig en default her? Fordi den switcher på ytelse, og så lenge alle ytelsene i enumet dekkes så er det ikke noe annet man kan sende inn og man trenger ikke en default? Usikker på om switch fungerer sånn, men føler det funker 🤠

Reply:
Det har du rett i! Ettersom man bare kan gi inn ytelser som finnes i enumen vil det jo være dekket når alle ytelsene er med i casene. Endret her: [e920576](https://github.com/bekk/nav-familie-endringsmelding/pull/90/commits/e920576ba7b169e041c0af27aff4e51a4f701561)